### DISCUSSION TITLE
Ore generation with custom world generator Plugin

### DISCUSSION THREAD

**Discussion by thisHexy** [DerFrZocker/Custom-Ore-Generator#16] [Q&A]
I am using Terra on my paper 1.19.3 server and am trying to figure out how to generate custom ores on its weld. In the description of Custom Ore Generator it says that custom world generators are supported.

In a vanilla world everything works as it should.

**Comment by DerFrZocker** [comment]
Custom Ore Generator does not supported custom biomes, which Terra makes use of. It only supportes Custom world generators which use, standard vanilla biomes.
